Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -808,8 +808,8 @@ YUI.add('ez-platformuiapp', function (Y) {
                     view: "serverSideView",
                     callbacks: ['open', 'checkUser', 'handleSideViews', 'handleMainView']
                 }, {
-                    name: "adminRole",
-                    regex: /\/admin\/(pjax%2Frole.*)/,
+                    name: "adminLanguage",
```

Comment:
Yes - but I just remembered that @mhyndle talked about needing two services. Mateusz, please verify that this was not something you added :)